webpages/register.html

```diff
@@ -0,0 +1,139 @@
+<!doctype html>
+<title>Username | Living Meta-Analysis</title>
+<script src="/lib/promise.js"></script>
+<script src="/lib/fetch.js"></script>
+<script src="https://apis.google.com/js/platform.js"></script>
+<link rel="stylesheet" href="/css/main.css">
+
+<header>
+  <a href="/" id="logo-box">
+    <h1>LiMA</h1><h2>Living Meta-Analysis</h2>
+  </a>
+</header>
+
+<h1>Please choose a username:</h1>
+<div class="register">
+  <p class="edithighlight">
+    <span class='customusername' contenteditable placeholder="custom username"></span>
+  </p>
+
+  <form class='suggestions'>
+  <h3>Suggestions:</h3>
+    <span id='emailaddress'> same as email address</span>
+    <span id='emailaddressfirst'> same as email address first part</span>
+    <span id='displayname'> same as Google display name (without spaces)</span>
+  <p>
+    <button class="confirm">confirm</button>
```

Comment:
change this button to register.

Remember to add:
- agree with this t&c's
- agree with this privacy policy: include a clear as day callout/textbox here which says "Your email address will be public"

register button only enabled if these 2 are ticked.